src/Extensions/Features/src/FeatureCollectionExtensions.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.AspNetCore.Http.Features;
+
+/// <summary>
+/// Extension methods for getting feature from <see cref="IFeatureCollection"/>
+/// </summary>
+public static class FeatureCollectionExtensions
+{
+    /// <summary>
+    /// Get feature of type <typeparamref name="TFeature"/> from the <see cref="IFeatureCollection"/>.
+    /// Exception of type <see cref="InvalidOperationException"/> thrown when asked for unregistered feature type
```

Comment:
```suggestion
    /// Retrieves the requested feature from the collection.
    /// Throws an <see cref="InvalidOperationException"/> if the feature is not present.
```

Using language similar to https://github.com/dotnet/aspnetcore/blob/main/src/Extensions/Features/src/IFeatureCollection.cs#L31-L36